DNN Platform/Library/Common/Globals.cs

```diff
@@ -2599,10 +2658,24 @@ public static TabType GetURLType(string URL)
         /// whether the internal links exist.
         /// If the link does not exist, the function will return an empty string.
         /// </summary>
-        /// <param name="ModuleId">Integer.</param>
-        /// <param name="url">String.</param>
+        /// <param name="moduleId">The id of the module (unused).</param>
+        /// <param name="url">the url to import.</param>
         /// <returns>If an internal link does not exist, an empty string is returned, otherwise the passed in url is returned as is.</returns>
-        public static string ImportUrl(int ModuleId, string url)
+        [Obsolete("Deprecated in 9.8.1, moduleId is not used in the method, use the overload that takes only the url, scheduled removal in v10")]

```

Comment:
Why is this being removed in v10 instead of v11?

Reply:
Addressed in latest commit, thanks.